test/stdlib/DispatchTypes.swift

```diff
@@ -27,14 +27,22 @@ if #available(OSX 10.10, iOS 8.0, *) {
 // dispatch/source.h
 _ = DispatchSource.makeUserDataAddSource()
 _ = DispatchSource.makeUserDataOrSource()
+#if !os(FreeBSD)
```

Comment:
How is this compiling on Linux right now? Surely whatever guards Linux should guard FreeBSD too.

Reply:
DispatchSource is basically kqueue wrapper, which exists on FreeBSD and Darwin but not on Linux

Reply:
Are we sure we want to commit to supporting it at this time, or does it make more sense to leverage future improvements aimed at Linux?

Reply:
I suspect that work to improve this on Linux would make use of Linux-specific functionality (epoll, say) that doesn't exist natively on FreeBSD